plotly_calplot/raw_heatmap.py

```diff
@@ -25,7 +33,11 @@ def create_heatmap_without_formatting(
             ygap=gap,  # and this is used to make the grid-like apperance
             showscale=False,
             colorscale=colorscale,  # user can setup their colorscale
-            hovertemplate="%{customdata[0]} <br>%{customdata[1]}=%{z} <br>Week=%{x}",
+            text=text,
+            hovertemplate=(
+                "%{customdata[0]} <br>Week=%{x} <br>%{customdata[1]}=%{z}"
```

Comment:
FYI: I also reordered this from  `<date> <value> <week>` to `<date> <week> <value>[ <text>]` (where text is optional) in order to keep the "date" parts together and avoid splitting `<value>` from `<text>`.